avr/cores/megacommand/Midi/MidiID.hh

```diff
@@ -7,6 +7,7 @@
 #define DEVICE_NULL 0xFF
 #define DEVICE_MIDI 0xFE
 #define DEVICE_MD 0x02
+#define DEVICE_MD 0x03
```

Comment:
DEVICE_MNM

Reply:
Also FYI MidiID does not work on the new machines last time I checked.

Reply:
If so we can use a MNM-specific sysex query as a probe.
Newer machine=digitakt etc?